src/index.ts

```diff
@@ -0,0 +1,6 @@
+require('dotenv').config();
+if (process.env.EXPRESS === 'true') {
```

Comment:
I think we should also add Nest and Fastify examples, especially Nest since it's setup is problematic for users (refering to Slack questions)

Reply:
I'll do it in the next iteration